app/controllers/entities_controller.rb

```diff
@@ -413,7 +415,7 @@ def upload_image
   private
 
   def set_entity_with_eager_loading
-    @entity = Entity.includes(:aliases, list_entities: [:list]).find(params[:id])
+    set_entity(includes: :aliases, list_entities: [:list])
```

Comment:
see above comment how how set_entity is being used here.

Reply:
+1